plugins/cilium-cni/cilium-cni.go

```diff
@@ -51,6 +52,10 @@ import (
 	"github.com/cilium/cilium/plugins/cilium-cni/types"
 )
 
+const (
+	defaultLogRetentionInDays = 7
```

Comment:
Argh, I really hate defaults like this, only because you never know the size of disk you're working with. 
Does the logging infra have a threshold which will flush the log file if disk space gets tight?

Reply:
To my knowledge, the logging rotation library (lumberjack.v2) we're using doesn't garbage collect based on disk pressure. 

But what we could do is set is `MaxBackups`. This will impose an upper bound on the total disk space, i.e. to `MaxSize * MaxBackups` megabytes, regardless of the retention duration.

See https://pkg.go.dev/gopkg.in/natefinch/lumberjack.v2#section-readme for details.

Reply:
thanks both, it's reasonable to add max backups setting to have upper bound here, the default configuration is to log only error, so hopefully, we will never reach such limit.

Reply:
just a note that with compression for old log, the actual disk size will be much less than MaxSize * MaxBackups.